Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -76,9 +78,17 @@ namespace SandboxEditor
                 AzFramework::WindowRequestBus::EventResult(
                     windowSize, event.m_windowHandle, &AzFramework::WindowRequestBus::Events::GetClientAreaSize);
 
-                const auto screenPoint = AzFramework::ScreenPoint(
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetX() * windowSize.m_width),
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetY() * windowSize.m_height));
+                if (m_virtualNormalizedPosition) {
+                    (*m_virtualNormalizedPosition) += position->m_normalizedPositionDelta;
+                } else {
+                    m_virtualNormalizedPosition =
+                        AZStd::optional<AZ::Vector2>{position->m_normalizedPosition};
+                }
```

Comment:
I believe this should also be wrapped in `SandboxEditor::ManipulatorMouseWrap()` now, we only want this logic if we explicitly have that setting enabled, otherwise things should behave the same as before.